src/styles/index.scss

```diff
@@ -1,3 +1,72 @@
+$box-size: 100px;
+$gap: 1px;
+$week-day: (
+  'mon': 0,
+  'tue': 1,
+  'wed': 2,
+  'thu': 3,
+  'fri': 4,
+  'sat': 5,
+  'sun': 6,
+);
+
 body {
   margin: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap;
+  padding: 0 10px;
+  max-width: 726px;
+
+  &__day {
+    position: relative;
+    background-color: #eee;
+    height: $box_size;
+    width: $box_size;
```

Comment:
Similarly, correct the typo in the width property to use `$box-size` instead of `$box_size`.